docs/volumes.md

```diff
@@ -0,0 +1,33 @@
+# Volumes
+
+In cases where multiple volumes are neccessary for your Jupyter notebook such as:
+- a set of users want to have access to a shared volume
+- a workshop requires some dataset/examples and instead of pulling it from S3, you want to have it mounted in the container
+- you want to provide additional storage to a set of users
+- specialized binaries/libraries mounted into the container
+it is possible to define volumes to be added to the notebook.
```

Comment:
There needs to be one empy line between the last item in the list and new paragraph, otherwise it "collapses"